posthog/hogql/database/database.py

```diff
@@ -153,6 +155,7 @@ class Database(BaseModel):
         "sessions",
         "heatmaps",
         "query_log",
+        "exchange_rate",
```

Comment:
You can query the `exchange_rate` table now! I think this will be a fun addition, no reason not to do it